pages/voting-guides/index.tsx

```diff
@@ -76,14 +92,40 @@ const VotingGuides: NextPage<{
             {votingGuides && votingGuides.length < 1 && (
               <small>No voting guides</small>
             )}
+
             {votingGuides?.map((guide) => (
               <VotingGuideCard
                 guide={guide as Partial<VotingGuideResult>}
                 key={guide.id}
+                showEdit={user.id === guide.user.id}
               />
             ))}
           </FlagSection>
         </div>
+        {!!savedGuides?.data?.votingGuidesByIds?.length && (
+          <div className={styles.votingContainer}>
+            <FlagSection title="Other Guides">
+              {/* This section uses {election} since we only have one right now.
+                As the app gets more dynamic this must be changed. */}
+              <div className={styles.votingHeader}>
+                <h1>{dateString(election?.electionDate)}</h1>
+                <h2>{election?.title}</h2>
+                <p>{election?.description}</p>
```

Comment:
If any of these fields can be empty, it would be better to conditionally render the tags to avoid extra vertical padding/ spacing when the content is empty.

Reply:
I don't think they will be, it's just Typescript being careful, but agreed it's better form

Reply:
I'm actually going to pull the election head out as its own component